security/backends/common/is_core.py

```diff
@@ -440,10 +462,12 @@ class CeleryTaskInvocationLogCoreMixin(CeleryCoreMixin, LogCoreMixin):
 
     menu_group = 'celerytaskinvocationlog'
 
-    rest_fields = list_fields = (
-        'id', 'start', 'stop', 'release', 'slug', 'celery_task_id', 'name', 'state_with_duplicate', 'time',
-        'short_input', 'queue_name'
+    ALL_FIELDS = (
+        'start', 'stop', 'time', 'name', 'state_with_duplicate', 'short_input', 'queue_name', 'is_duplicate',
+        'celery_task_id', 'release'
     )
+    rest_fields = ('id', *ALL_FIELDS, 'slug')
```

Comment:
why not rest_fields = list_fields + ('id,)

why you need the slug in rest_fields?

Reply:
`slug` used to be in rest_fields before so I didn't want to break anything.
If I do `rest_fields = list_fields + ('id,)` wouldn't be `short_id` redundant in rest_fields? Because I need to have it in `list_fields`.

Reply:
so you should firstly check if something is not brokern. I do not see the sense

Reply:
for us your solution also works (no issues found) so I changed that.